general/clients/codec-support.md

```diff
@@ -88,31 +90,54 @@ Subtiles can be a subtle issue for transcoding. Containers have a limited number
 |VobSub<sup>3</sup>|Picture|✅|✅|✅|🔶|
 |MP4TT/TXTT|XML|❌|✅|❌|❌|
 |PGSSUB|Picture|❌|❌|✅|❌|
+|EIA-608/708<sup>4</sup>|Embedded|✅|✅|✅|❌|
 
 <sup>1</sup>MP4 containers can only support one embedded subtitle stream. This does not affect external subtitles.
```

Comment:
The "one subtitle per MP4 container" thing doesn't seem right. I've been able to find multiple ffmpeg commands that allow you to put two or more. No idea if that breaks the spec, though.

Reply:
it might be that only MP4TT/3GPP are supported. cant really find much about the number of subs allowed.